content/20.treatments.md

```diff
@@ -105,6 +105,49 @@ Summarize the state of the antiviral approach.
+These include vitamins and minerals along with extracts and omega-3 polyunsaturated fatty acids (n-3 PUFA) [@doi:10.1002/jmv.25707].
+Considerable evidence in vitro and in vivo suggests that nutraceuticals containing phycocyanobilin, N-acetylcysteine, glucosamine, selenium or phase 2 inductive nutraceuticals (e.g. ferulic acid, lipoic acid, or sulforaphane) can prevent or modulate RNA virus infections via amplification of the signaling activity of mitochondrial antiviral-signaling protein (MAVS) and activation of toll-like receptor 7 (TLR7) [@doi:10.1016/j.pcad.2020.02.007].
+
+Another potential nutraceutical that has exhibited beneficial effects against various viral infections is n-3 PUFA [@doi:10.1002/jmv.25707], such as eicosapentaenoic acid (EPA) and docosahexaenoic acid (DHA).
+N-3 PUFA nutraceuticals can mediate inflammation and they have the capacity to modulate the adaptive immune response [@doi:10.1111/j.1365-2125.2012.04374.x; @doi:10.3390/molecules22111964; @doi:10.1038/nrcardio.2016.103].
+One potential mechanism by which n-3 PUFA could exert beneficial effects against viral infections is by acting as precursor molecules for the biosynthesis of endogenous specialised proresolving mediators (SPM) like protectins and resolvins that actively resolve inflammation and infection [@doi:10.1016/j.immuni.2014.02.009]. 
+SPM have exhibited beneficial effects against a variety of lung infections including RNA viruses [@doi:10.1038/nri.2015.4].
+Indeed, protectin D1 has been shown to increase survival from H1N1 viral infection in mice by affecting the viral replication machinery [@doi:10.1016/j.cell.2013.02.027].
+Overall, there is currently limited evidence that these nutraceuticals may affect COVID-19 infection as there is a significant lack of human studies.  
+
+In contrast, there is evidence that nutrient supplements may exhibit some benefit against RNA viral infections. 
+Zinc is a trace metal obtained from dietary sources or supplementation that is important for the maintenance of immune cells involved in adaptive and innate immunity [@doi:10.1016/j.abb.2016.03.022].
+Indeed, adequate zinc intake has been associated with reduced incidence of infection [@doi:10.1093/ajcn/85.3.837] and antiviral immunity [@doi:10.1093/advances/nmz013].
+Clinical trial data supports the utility of zinc to diminish the duration and severity of symptoms associated with common colds when it is provided within 24 hours of the onset of symptoms [@doi:10.1331/1544-3191.44.5.594.Hulisz; @doi:10.2174/1874306401105010051].
+In coronaviruses specifically, in vitro evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015]. 
```

Comment:
From looking into it, it seems like SARS-CoV-GFP is just a fluorescently tagged version of SARS-CoV, but I initially assumed it was a different strain -- it might be worth clarifying in the text
I'm curious whether there's an association between nutrition status and intracellular zinc levels? I assume we don't know whether zinc supplementation or diet modulates zinc concentration at the cellular level?

Reply:
Good question, there is limited evidence from what I have read. From what I have read previously it is specific to the target organ. I will look into this

Reply:
So yes, nutritional status intracellular zinc levels are linked does - Examples of zinc supplementation increasing plasma zinc levels and other tissues in uremia patients (https://academic.oup.com/ajcn/article-abstract/36/6/1177/4693588). 

Indeed, it is well covered in infection, for example: https://academic.oup.com/ajcn/article/85/3/837/4633003

I will add this to the section as it is a very interesting point to make.